LICENSE

```diff
@@ -1,14 +1,206 @@
-Copyright (c) 2024, Heroku <>
+Apache License Version 2.0
 
-Permission to use, copy, modify, and/or distribute this software for any
-purpose with or without fee is hereby granted, provided that the above
-copyright notice and this permission notice appear in all copies.
+Copyright (c) 2024 Salesforce, Inc.
```

Comment:
Should this be updated to 2025?

Reply:
Copyright dates represent the year of publication and thus the first year in which these protections are active. It is not necessary and often ill-advised to change this as it will require legal approval to do so.